templates/artist/artist_detail.html

```diff
@@ -36,136 +36,86 @@
-            {% if investors %}
-                <li class="tabs-title"><a href="#investors">Investors</a></li>
-            {% endif %}
-            {% if artist.past_campaigns %}
-                <li class="tabs-title"><a href="#past">Past Campaigns</a></li>
-            {% endif %}
-            {% if updates or has_permission_to_submit_update %}
-                <li class="tabs-title {% if form.errors %}is-active{% endif %}"><a href="#updates">Updates</a></li>
-            {% endif %}
-        </ul>
-        <div class="tabs-content" data-tabs-content="artist-detail-tabs">
-            {% include "artist/includes/artist_detail_overview.html" %}
-            {% if investors %}
-                {% include "artist/includes/artist_detail_investors.html" %}
-            {% endif %}
-            {% if artist.past_campaigns %}
-                {% include "artist/includes/artist_detail_past.html" %}
-            {% endif %}
-            {% if updates or has_permission_to_submit_update %}
-                {% include "artist/includes/artist_detail_updates.html" %}
-            {% endif %}
-        </div>
-    {% else %}
+    <h2>{{ artist.name }}</h2>
+    <ul class="tabs" data-tabs id="artist-detail-tabs">
+        <li class="tabs-title {% if not form.errors %}is-active{% endif %}"><a href="#overview" aria-selected="true"><i class="fa fa-user fa-2x tab-icon" aria-hidden="true"></i><span class="artist-tab">Overview</span></a></li>
+        <li class="tabs-title"><a href="#invest"><i class="fa fa-money fa-2x tab-icon" aria-hidden="true"></i><span class="artist-tab">Invest</span></a></li>
+        <li class="tabs-title"><a href="#projects"><i class="fa fa-pie-chart fa-2x tab-icon" aria-hidden="true"></i><span class="artist-tab">Projects</span></a></li>
+        <li class="tabs-title {% if form.errors %}is-active{% endif %}"><a href="#updates"><i class="fa fa-newspaper-o fa-2x tab-icon" aria-hidden="true"></i><span class="artist-tab">Updates</span></a></li>
+    </ul>
+    <div class="tabs-content" data-tabs-content="artist-detail-tabs">
         {% include "artist/includes/artist_detail_overview.html" %}
-    {% endif %}
+        {% include "artist/includes/artist_detail_invest.html" %}
+        {% include "artist/includes/artist_detail_projects.html" %}
+        {% include "artist/includes/artist_detail_updates.html" %}
+    </div>
 {% endblock %}
 
 {% block extrajs %}
     <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
+    <script type="text/javascript" src="{% static "js/vendor/Chart.min.js" %}"></script>
     <script src="https://checkout.stripe.com/checkout.js"></script>
-    <script type="text/javascript" src="{% static "js/Chart.js" %}"></script>
     <script type="text/javascript">
         var perdiem_fee = {{ PERDIEM_FEE }};
         var stripe_percentage = {{ STRIPE_PERCENTAGE }};
         var stripe_flat_fee = {{ STRIPE_FLAT_FEE }};
         var artist_name = '{{ artist.name|escapejs }}';
-
-        {% if campaign %}
```

Comment:
You've removed the `{% if campaign %}` statement around the campaign logic here in the JS, but that needs to be there. Without it, pages without a currently-running campaign will result in an error 500.